lsp/nls/src/requests/formatting_external.rs

```diff
@@ -17,8 +17,8 @@ pub fn handle_format_document(
     server: &mut Server,
 ) -> Result<(), ResponseError> {
     let path = uri_to_path(&params.text_document.uri)?;
-    let file_id = server.cache.id_of(&SourcePath::Path(path)).unwrap();
-    let text = server.cache.files().source(file_id).clone();
+    let file_id = server.world.cache.id_of(&SourcePath::Path(path)).unwrap();
```

Comment:
those build failures were only revealed when trying to build without the format feature, which doesn't happen outside of crates.io releases.